internal/sqldb/createtables.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqldb
+
+import (
+	"database/sql"
+)
+
+// CreateTables creates tables in MySQL database if they are not present.
+func CreateTables(sqlClient *sql.DB) error {
```

Comment:
I thought mixer can deal with the scenario where the tables don't exist yet.

Let's discuss this more. This creates a tight coupling between RSI and Mixer and will require keeping both in sync all the time.

Reply:
This is a fallback safe check. It only creates table if they do not exist to prevent query errors. RSI can create tables, then this will be a no op.

Reply:
RSI creates tables only if they don't exist. Also, creating them in 2 places can create maintenance issues.

Let's discuss if there is a cleaner / simpler option.